src/lib/accessibility/Behaviors/Tree/treeBehavior.ts

```diff
@@ -0,0 +1,15 @@
+import { Accessibility } from '../../types'
+
+/**
+ * @specification
+ * Adds role='tree' to the tree element and role='group' to the subtree elements.
+ */
+const treeBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: props.nested ? 'group' : 'tree',
```

Comment:
please do not use group and tree roles as they do not work correctly with some of the screen readers. Instead, follow the HTML structure that was sent before